ocs_ci/deployment/deployment.py

```diff
@@ -424,23 +424,10 @@ def deploy_ocs_via_operator(self):
             if local_storage and platform == 'aws':
                 resources = {
                     'mds': {
-                        'limits': {'cpu': 3},
-                        'requests': {'cpu': 1}
-                    },
-                    'noobaa-core': {
-                        'limits': {'cpu': 2, 'memory': '8Gi'},
-                        'requests': {'cpu': 1, 'memory': '8Gi'}
-                    },
-                    'noobaa-db': {
-                        'limits': {'cpu': 2, 'memory': '8Gi'},
-                        'requests': {'cpu': 1, 'memory': '8Gi'}
+                        'limits': {'cpu': '3'},
```

Comment:
I guess this has changed based on version right? In 4.4 documentation we have defined that we need to specify all what was before, or did it change for 4.4 as well?

If so, this has to be dependent on the version and in some version to do one thing and in other to do other things when generating those limits based on the documentation.

Reply:
This I'm unsure of, I was just going based off of @nehaberry 's comment above. Maybe she can weigh in here?

I do think you're right here though. Since we're doing multiple configurations differently based on version as well as whether or not we're configured for production I think we need to pull this code out into it's own function to try to organize.

Reply:
I've updated and have left the limits for `noobaa-db` and `noobaa-core` for ocs_versions < 4.5. Regarding `noobaa-endpoint` I have removed setting those limits entirely because they were being done for ocs_verions >= 4.5 and we are removing noobaa limits being set for 4.5 and beyond. If this assumption is incorrect I can add those limits back in.